tests/compile.sh

```diff
@@ -167,7 +171,7 @@ if [[ "${MAKE_OPT}" == *"S2S=Y"* ]]; then
     CMAKE_FLAGS="${CMAKE_FLAGS} -DS2S=Y"
 fi
 
-if [[ "${MAKE_OPT}" == *"DATM=Y"* ]]; then
+if [[ "${MAKE_OPT}" == *"DATM=Y"* ]] && [[ "${MAKE_OPT}" != *"CDEPS_DATM=Y"* ]]; then
```

Comment:
This is going to be confusing when adding future data atmosphere components but I don't have a solution at the moment.

Reply:
@danrosen25 yes I agree. It requires to improve the build system which takes little bit time.